### DISCUSSION TITLE
REQ: different projection types aka playable fov 180

### DISCUSSION THREAD

**Discussion by mushis** [ezQuake/ezquake-source#492] [Ideas]
FTE has a really cool feature that is the support for multiple projection types, for example Steregraphic, fisheye, panoramic, among others.
**This enables playable fov 180**, which for ultrawide monitors is a must. It's really cool, even for 16:9 monitors, i suggest you to try it (get AfterQuake from FTE website)

**Is your feature request related to a problem? Please describe.**
no. only that fov from 130 or 140 is unplayable in all quake engines, except the ones that have different projection types.

**Describe the solution you'd like**
copy/paste whatever FTE has ;)

**Describe alternatives you've considered**
nop

**Additional context**
![image](https://user-images.githubusercontent.com/3011167/107023910-331e0b00-679f-11eb-8aaf-db4fbc9db409.png)
fov 150, panini projection. this screenshot was taken 5 years ago, projection support in FTE is better now.

**Impact on server or mods**
none

EDIT: Attached a zip with screenshots (comparison between different projection types)
[screenshots.zip](https://github.com/ezQuake/ezquake-source/files/5933687/screenshots.zip)

**Comment by meag** [comment]
Moved to discussions: https://github.com/ezQuake/ezquake-source/discussions/491

**Comment by igaxokihowu903-commits** [comment]
Capisco perfettamente cosa intendi: oltre un certo FOV, tipo 130–140, la proiezione classica diventa davvero scomoda e distorta, soprattutto su monitor ultrawide. Il supporto a diversi tipi di proiezione, come in FTE, renderebbe l’esperienza [allinbet-it.com](https://allinbet-it.com/) molto più naturale e godibile, anche con FOV elevati come 180. Non è solo una questione estetica, ma proprio di giocabilità. Avere opzioni come fisheye o panoramica darebbe più libertà all’utente senza rompere il feeling del gioco. Sarebbe una bella aggiunta, anche se implementarla non è banale.

**Comment by EvanGoodwin22** [comment]
Ναι, καταλαβαίνω τι λες και γιατί το θεωρείς σημαντικό. Το FTE έχει πράγματι πολύ δυνατό σύστημα με διαφορετικούς τύπους προβολής όπως stereographic, fisheye και panoramic, και αυτό κάνει τα πολύ υψηλά FOV (π.χ. 180) πραγματικά playable, ειδικά σε ultrawide οθόνες. Σε άλλα Quake engines το FOV πάνω από 120–140 συχνά γίνεται [monster-wins.gr](https://monster-wins.gr/) παραμορφωτικό και δύσχρηστο, οπότε η εμπειρία πέφτει πολύ. Η ιδέα σου ουσιαστικά είναι να μεταφερθεί αυτούσιο αυτό το projection system, γιατί λύνει ένα πραγματικό usability θέμα και όχι απλά αισθητικό.

**Comment by JesseReed123** [comment]
FTE má opravdu skvělou funkci podporující různé typy projekce, například stereografickou, rybí oko nebo panoramatickou. To umožňuje hrát s FOV 180, což je u ultrawide monitorů skoro nutnost. I na klasických 16:9 monitorech je to [roobet ](https://roobetcasino-cz.cz/)velmi příjemné. Moje žádost je vlastně jednoduchá – chtěl bych, aby se tato funkce zkopírovala nebo implementovala do ostatních Quake enginů. Alternativy jsem moc nezvažoval, protože jiné řešení prostě neexistuje.
